osu.Game/Migrations/20190525060824_SkinSettings.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace osu.Game.Migrations
+{
+    public partial class SkinSettings : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "SkinInfoID",
+                table: "Settings",
+                nullable: true);
+
+            migrationBuilder.CreateIndex(
+                name: "IX_Settings_SkinInfoID",
+                table: "Settings",
+                column: "SkinInfoID");
+
+            // unsupported by sqlite
```

Comment:
So why is this here then? How are we using foreign keys everywhere else just fine but this one doesn't work? Why did the migration auto-generate this one here and not in all the other cases?

Reply:
as discussed in real life (all previous foreign key additions were performed along with a `CreateTable`).

i'll see if i can do a manual migration this time.